### PR TITLE
USB redirection is not supported by pseries

### PR DESCRIPTION
**Pull request by chloerh**

### CODE REVIEW

**Comment by chloerh** [comment]
@dzhengfy could you confirm this?

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_detach_device_alias.cfg [hunk 1]
```diff
@@ -12,7 +12,7 @@
             detach_controller_mode = "virtio-scsi"
             detach_check_xml = "<controller type='scsi' index='%s' model='virtio-scsi'>"
         - redirdev:
-            no s390-virtio
+            no s390-virtio pseries
```
Comment (smitterl) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_detach_device_alias.cfg]: I wondered if a comma was missing here but double checked with `avocado list -V ... --vt-machine-type pseries`, resp. `s390-virtio` and `q35` and filter worked correctly.

**Comment by dzhengfy** [comment]
> @dzhengfy could you confirm this?

@chloerh , could you help add the error message which indicates unsupported feature next time?

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_detach_device_alias.cfg (hunk 1 above)
Comment (dzhengfy) [reply libvirt/tests/cfg/virsh_cmd/domain/virsh_detach_device_alias.cfg]: @smitterl , comparing with the output on x86_64, 'usb-redir' is for USB redirection. So it is not supported on PPC.

```
# /usr/libexec/qemu-kvm -device help|grep -i usb
name "usb-host", bus usb-bus
name "usb-hub", bus usb-bus
USB devices:
name "ich9-usb-ehci1", bus PCI
name "ich9-usb-ehci2", bus PCI
name "ich9-usb-uhci1", bus PCI
name "ich9-usb-uhci2", bus PCI
name "ich9-usb-uhci3", bus PCI
name "ich9-usb-uhci4", bus PCI
name "ich9-usb-uhci5", bus PCI
name "ich9-usb-uhci6", bus PCI
name "nec-usb-xhci", bus PCI
name "piix3-usb-uhci", bus PCI
name "piix4-usb-uhci", bus PCI
name "usb-ehci", bus PCI
name "vt82c686b-usb-uhci", bus PCI
name "usb-bot", bus usb-bus
name "usb-storage", bus usb-bus
name "usb-ccid", bus usb-bus, desc "CCID Rev 1.1 smartcard reader"
name "usb-kbd", bus usb-bus
name "usb-mouse", bus usb-bus
name "usb-tablet", bus usb-bus
name "usb-redir", bus usb-bus
```

Comment (smitterl) [reply libvirt/tests/cfg/virsh_cmd/domain/virsh_detach_device_alias.cfg]: @dzhengfy Got it. Thank you for clarifying :D

**Comment by chloerh** [comment]
> > @dzhengfy could you confirm this?
> 
> @chloerh , could you help add the error message which indicates unsupported feature next time?

Sure

**Pull request merged**